src/styles/blocks/content.scss

```
@@ -0,0 +1,20 @@
+.main {
+  display: flex;
+  justify-content: center;
+
+  &__wrapper {
+    display: flex;
+    flex-wrap: wrap;
+    max-width: whatIsContentWidth($card-width, $card-gap, $one-line);
+    align-items: baseline;
+    margin-top: 50px;
```

```suggestion
    margin: 50px 0;
```
Краще зробити нижній відступ, щоб картки не прилягали плотно до низу.